swan-lake/learn/security/writing-secure-ballerina-code.md

```diff
@@ -95,8 +98,7 @@ There can be certain situations where a tainted value must be passed into a secu
 // Execute select query using the untrusted (tainted) student ID
 boolean isValid = isNumeric(studentId);
 if (isValid) {
-   var dt = testDB->select("SELECT NAME FROM STUDENT WHERE ID = " +
-                           <@untainted> studentId, ResultStudent);
+    var resultStream = jdbcClient->query("SELECT NAME FROM STUDENT WHERE ID = " + <@untainted>studentId);
```

Comment:
testDB var is not there in example started line 60. So I add the line 66 with <untainted> variable.